sormas-api/src/main/java/de/symeda/sormas/api/event/EventExportDto.java

```diff
@@ -278,6 +284,16 @@ public EventSourceType getSrcType() {
 	}
 
 	@Order(19)
+	public InstitutionalPartnerType getSrcInstitutionalPartnerType() {
```

Comment:
Unify this and the details field below and build an appropriate string directly in the constructor (see CaseExportDto.educationType for an example)